### DISCUSSION TITLE
pyarrow not installed

### DISCUSSION THREAD

**Discussion by ravi-jam** [cuebook/CueObserve#187] [Q&A]
Hi Folks, 

My team just deployed CueObserve to EC2. I Got this error when we tried to add a dataset 

>  {“log”:“2021-10-16 06:15:14,232 [dbConnections.bigquery:63] ERROR  Can’t connect to db with this credentials The pyarrow library is not installed, please install pyarrow to use the to_arrow() function.\n”,“stream”:“stderr”,“time”:“2021-10-16T06:15:14.237031452Z”} 

Any help would be appreciated.

**Comment by vincue** [comment]
Which version of CueObserve are you on?

**Reply by vincue** [reply]
Yeah, thanks for reporting. Will fix it in a day or two & let you know.

**Comment by sachinkbansal** [comment]
@vincue created issue https://github.com/cuebook/CueObserve/issues/188 for this

**Comment by ravi-jam** [comment]
> Which version of CueObserve are you on?

Where can I check the version number :)

**Reply by ArunMu21** [reply]
Hello @ravi-jam , Did you find any solution for above issue? I am also getting the same error while I am connecting to BigQuery Dataset.
